statsmodels/tsa/arima/params.py

```diff
@@ -351,6 +575,86 @@ def is_invertible(self):
 
         return ma_stationary and seasonal_ma_stationary
 
+    def set_fixed_params(self, fixed_params, validate=False):
```

Comment:
Just curious: why default `validate=False`?

Reply:
Ah, I was imagining that, in most cases, we probably have separately checked `fixed_params` with `validate_fixed_params` along with any other checks before we call `set_fixed_params`, so usually there may not be the need to validate again.

Reply:
Sounds good to me, I'll leave the final decision up to you.